tools/docker/README.md

```diff
@@ -4,77 +4,169 @@ The docker-compose configuration present in this directory allows for a user to
+$$
+$$ cldev core # import our testing key and api credentials, then start the node
+# 
+# ** Importing default key 0x9ca9d2d5e04012c9ed24c0e513c9bfaa4a2dd77f
+# 2019-12-11T20:31:18Z [INFO]  Locking postgres for exclusive access with 500ms timeout orm/orm.go:74        #   
+# 2019-12-11T20:31:18Z [WARN]  pq: relation "migrations" does not exist           migrations/migrate.go:149 
+# ** Running node
+# 2019-12-11T20:31:20Z [INFO]  Starting Chainlink Node 0.7.0 at commit 7324e9c476ed6b5c0a08d5a38779d4a6bfbb3880 cmd/local_client.go:27  
+# 2019-12-11T20:31:20Z [INFO]  SGX enclave *NOT* loaded                           cmd/enclave.go:11       
+# 2019-12-11T20:31:20Z [INFO]  This version of chainlink was not built with support for SGX tasks cmd/enclave.go:12       
+# ...
+# ...
 ```
 
-### Build a single image
-The following command will build a single image, along with all of its dependent images.
+In a new terminal:
 ```sh
-docker-compose build <service name>
+docker logs operator-ui -f
 ```
+You'll now have two terminals, one with the core node, one with operator-ui, with both being able to react to code changes without rebuilding their respective images.
 
-## Run
-Running a service is the process of launching a process for a built image, allowing us to interact with it. Docker compose lets up run multiple interdependent services in a fashion where they are launched in the correct order.
-
-### Run all services
-The following command will run up all services and their dependent child services.
-```sh 
-docker-compose up
+### Running integration test suites
+The integration test suite will run against a parity node by default. You can run the integration test suite with the following command:
+```sh
+$ ./compose test
 ```
 
-Adding the `-d` flag will detach the spun up services from your terminal.
-```sh 
-docker-compose up -d
+If you want to run the test suite against a geth node, you can set the `GETH_MODE` environment variable.
+```sh
+$ GETH_MODE=true ./compose test
 ```
 
-### Run a single service
-The following command will run up a single service along with any service dependencies it has.
+If we want to quickly test new changes we make to `integration/` or `tools/ci/ethereum_test` without re-building our images, we can use the `test:dev` command which will reflect those changes from the host file system without rebuilding those containers.
 ```sh
-docker-compose up <service name>
+$ ./compose test:dev
 ```
 
-Adding the `-d` flag will detach the spun up services from your terminal.
+### Cleaning up
+To remove any containers, volumes, and networks related to our docker-compose setup, we can run the `clean` command:
 ```sh
-docker-compose up -d
+./compose clean
 ```
 
-## Teardown
-### Shutdown running services
+### Running your own commands based off of docker-compose
+The following commands allow you do just about anything:
 ```sh
-docker-compose down
+$ ./compose <subcommand>
+$ ./compose integ <subcommand>
```

Comment:
What is an `integ`?

Reply:
Integration

Reply:
https://github.com/smartcontractkit/chainlink/pull/1961/files#diff-4c50d5d1cec9105f53b377d8904042b4R38

Reply:
It's a bit of a weird abbreviation, could we add a full label? Maybe `i` for short as an alternative.

Reply:
9f5518664afd83fd1c0bf84bf97783ea3c1bc781